doc/ko/uftrace-record.md

```diff
@@ -148,6 +148,11 @@ RECORD 옵션
     으로 '추적' trigger가 적용된 경우, 그 함수는 실행 시간과 상관없이 항상
     출력된다. 필터에 대한 설명은 *FILTERS* 를 참고한다.
 
+-Z *SIZE*, \--size-filter=*SIZE*
+:   SIZE 바이트보다 큰 함수들을 표시하지 않게 한다. 만약 어떤 함수가 명시적으로
+    '추적' trigger가 적용된 경우, 그 함수는 함수 크기와 상관없이 항상 출력된다.
```

Comment:
Please do not translate `'trace'` as it's the name of the trigger action and needs to be specified in the command line as is.